packages/edit-site/src/components/global-styles/header.js

```diff
@@ -12,22 +12,26 @@ import {
 import { isRTL, __ } from '@wordpress/i18n';
 import { chevronRight, chevronLeft } from '@wordpress/icons';
 
-function ScreenHeader( { title, description } ) {
+function ScreenHeader( { title, description, showBack = true } ) {
```

Comment:
Rather than making this a prop, could we consider providing `<NavigatorToParentButton>` as a child of `<ScreenHeader>` and then just outputting whatever child components are included. This avoids having extra props, and keeps the concerns separated.

Reply:
I think a better way to handle this is probably to add something to the context so that we can use that to determine whether to show the back button.

Reply:
Actually even using context is tricky, because the back button is dependent on two different conditions:

1. Is it in the Site View sidebar?
2. Is it used in a root level component

Given this, I wonder if its simpler to just pass props, at least for a first iteration